experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveTable.java

```diff
@@ -72,17 +74,18 @@
   public HiveTable(HiveMetaStoreClient hive, HiveDatabase hiveDatabase,
       PathTranslator pathTranslator, String name, Schema schema, String baseLocation,
       List<ColumnStatisticsInfo> statistics, List<FieldSchema> cols, List<Partition> partitions,
-      Table table) throws IOException {
-    // TODO(gpang): don't throw exception in constructor
+      Table table) {
     mHive = hive;
     mHiveDatabase = hiveDatabase;
+    mTable = table;
     mPathTranslator = pathTranslator;
+    mPartitions = partitions;
+    // The following field are always populated
```

Comment:
I'm not sure what this means? the other fields also look populated?

Reply:
oops this comment is outdated. will remove